File: HelloWorld.java
public class HelloWorld {
  	public static void main(String... args) {
  		if (args.length > 0) {
  			util.cmd.Console.writeLine("Hello ", args[0]);
  		} else {
  			util.cmd.Console.writeLine("Hello World");
  		}
  	}
}

File: HelloWorld.php
<?php

class HelloWorld extends \lang\Object {
  public static function main($args) {
  	if (sizeof($args) > 0) {
      \util\cmd\Console::writeLine('Hello ', $args[0]);
    } else {
      \util\cmd\Console::writeLine('Hello World');
    }
  }
}